docs/guides/kubernetes/enable-lke-high-availability/index.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: enable-lke-high-availability
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Learn how to enable High Availability on Linode Kubernetes Engine.'
+og_description: 'Enable High Availability on Linode Kubernetes Engine'
+keywords: ['kubernetes', 'lke', 'high availability', 'ha']
+tags: ["networking","kubernetes","container"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-11-04
+modified_by:
+  name: Linode
+title: "How to Enable High Availability for LKE Clusters"
+h1_title: "Enabling High Availability for LKE Clusters"
+_build:
+  list: false
+noindex: true
+---
+
+
+## Enable High Availability
+
+In LKE, enabling HA ([High Availability](/docs/guides/introduction-to-high-availability/)) will create additional hardware nodes and replicate [control plane components](/docs/guides/beginners-guide-to-kubernetes-part-2-master-nodes-control-plane/), adding additional points of failure for your Kubernetes Cluster and decreasing the chance of any potential downtime. HA is an optional feature recommended for production workloads that must be enabled manually either when creating a new cluster, or by editing a pre-existing cluster.
```

Comment:
```suggestion
In LKE, enabling HA ([High Availability](/docs/guides/introduction-to-high-availability/)) will create additional replicas of your [control plane components](/docs/guides/beginners-guide-to-kubernetes-part-2-master-nodes-control-plane/), adding additional points of failure for your Kubernetes Cluster and decreasing the chance of any potential downtime. HA is an optional feature recommended for production workloads that must be enabled manually either when creating a new cluster, or by editing a pre-existing cluster.
```
it creates additional replicas, but not additional hardware. also, this wording sounds a bit negative to me: `adding additional points of failure` but tbh i dont have a better suggestion

Reply:
Good point, I think `adding an additional layer of redundancy` is a better way of wording it